doc/man1/dgst.pod

```diff
@@ -2,13 +2,14 @@
 
 =head1 NAME
 
-dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5, blake2b, blake2s - message digests
+dgst
+- generate or verify message digests
```

Comment:
I find "or verify" misleading. dgst does verification, but *signatures*. As far as plain digests go, it only generates digest.